Makefile

```diff
@@ -234,6 +234,8 @@ unittest:
 .PHONY: verify-mocks
 verify-mocks: mocks
 	@if !(git diff --quiet HEAD); then \
+		git status \
+		git diff \
```

Comment:
Can remove these now?
```suggestion
```

Reply:
I think they are useful so we can see what files are not matching on future failures